### ISSUE TITLE
Have 2 view counters: views and viewers

### ISSUE THREAD

**Issue by souldessin** [Chocobozzz/PeerTube#3796]
First off, thanks so much for the great system. I'm using it for my church and it has really made things better for providing a great alternative.

I would like to propose the feature of allowing the keeping the views counter max number as the starting number of views when a live stream is ended. There are those in my church that don't understand technology and think that no one is watching because they check the numbers when they get home. They don't understand that we have a dozen people tune in during the service and then the counter is reset.

**tldr:** A setting in administration that allowed the keeping of the live stream views when the video is converted to a normal video would be great!

**Comment by Chocobozzz** [comment]
We should have 2 distinct views counter:
 * for viewers (for now they are used by lives, but it could also be interesting for vod videos)
 * for views (and so we could take into account live views, like we do for vod videos)
